### DISCUSSION TITLE
Studio-Elf Announcements Module released 3.4.0

### DISCUSSION THREAD

**Discussion by leigh-pointer** [oqtane/oqtane.framework#2709] [Show and tell]
This is a maintenance release and a needed update if you wish to use with the Framework version 3.4.0+

**Comment by vnetonline** [comment]
I was just trying your announcements module but the tags don't seem to work

**Comment by leigh-pointer** [comment]
Will look into it!
Can you tell me what versions you are running; oqtane and announcements module?
________________________________

**Comment by vnetonline** [comment]
I am running Oqtane 3.4.3 and below is your modules installed on my website [amazingsolutions.com.au](https://amazingsolutions.com.au)

![image](https://user-images.githubusercontent.com/15242923/236362310-356c2a79-8fdd-4edb-b4fd-903911b98bca.png)


![image](https://user-images.githubusercontent.com/15242923/236362187-b936e8c3-2fbf-45a3-b23d-17a5112fed4a.png)

Also I don't see the Tagit tables 

![image](https://user-images.githubusercontent.com/15242923/236362339-1a2a7946-7b31-4833-b330-20c97d79b63e.png)

**Reply by leigh-pointer** [reply]
@vnetonline A new version is now in the Marketplace.  

You should check out the DynamicForm Module. On my site there are quite a few templates for the module, fully working and one of them is an Articles module which is very nice.  I would say about 80% of Studio-elf.net is created using DynamicForm.

Anyway, let me know how you get on.

**Comment by vnetonline** [comment]
@leigh-pointer still not working i upgraded to new version

**Reply by leigh-pointer** [reply]
Browser?
I have it installed on my site with no issues

**Comment by vnetonline** [comment]
Cleared the Browser and here is what happens 

![StudioElfAnnouncements-TestingTag](https://user-images.githubusercontent.com/15242923/236963088-3cbca9f9-267e-438b-b0c4-bacaf0be37ed.gif)

**Reply by leigh-pointer** [reply]
enter the tag then press enter, you can enter more that one tag.

![Tags](https://user-images.githubusercontent.com/9447612/237058880-4f7d96a6-f6b7-408e-a026-976db65ae508.gif)

**Reply by leigh-pointer** [reply]
@vnetonline Is this issue for you now fixed, or knowledge transfered?

**Reply by vnetonline** [reply]
Yes that works great maybe should be added in the help text that you have to press enter to make it a tag ... if you just type and press submit it doest work

![image](https://github.com/oqtane/oqtane.framework/assets/15242923/677d9ab0-2d72-4b1d-8c8e-538763966d24)
